core/src/main/java/com/redhat/lightblue/client/request/AbstractLightblueDataRequest.java

```diff
@@ -45,6 +49,15 @@ protected void appendRangeToJson(ObjectNode node, Integer begin, Integer maxResu
         }
     }
 
+    public void execution(Execution execution) {
+        this.execution = execution;
```

Comment:
Just a comment really, but other methods on the data request objects return themselves for chaining purposes. Not a requirement, but might be nice.